felix/fv/bpf_attach_test.go

```diff
@@ -92,4 +93,71 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ Felix bpf reattach object",
 			return out
 		}, "15s", "1s").ShouldNot(ContainSubstring("eth0"))
 	})
+
+	It("should attach programs to the bond interfaces", func() {
```

Comment:
btw can this be tested in the unit test in bpf_ep_mgr_test.go? Woudn't that be a better place? Perhaps those cases with slave in regexp and master not in addition to this test? There is a mock for netlink.